modules/rpc-util.js

```diff
@@ -23,6 +25,20 @@ class RpcRequest {
 
     static checkNodeStatus(provider) {
         return new Promise(function(resolve, reject) {
+            if(process.platform.includes(WIN_PROCESS_PLATFORM)) {
+                let current_docker_machine_ip;
+                try { 
+                    current_docker_machine_ip = docker_machine_ip();
+                } catch(error) {
+                    reject(error);
```

Comment:
pls test this scenario where docker_machine_ip fails and see if it logs/shows correct information.

Reply:
It didn't seemed to get failed as in case there is no running docker-machine detected on system it returns the value - "localhost".